src/test/java/marquez/api/JobResourceTest.java

```diff
@@ -122,10 +122,9 @@ public void testCreateJobBadInputs() throws MarquezServiceException {
     when(MOCK_NAMESPACE_SERVICE.get(any())).thenReturn(Optional.of(Generator.genNamespace()));
 
     JobResponse jobForJobCreationRequest = generateApiJob();
-    jobForJobCreationRequest.setLocation(null);
 
     Response res = insertJob(jobForJobCreationRequest);
-    assertEquals(UNPROCESSABLE_ENTRY_STATUS_CODE, res.getStatus());
+    assertEquals(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), res.getStatus());
```

Comment:
Can we do a static import here?

Reply:
Yeah, I thought about this! But, all status codes are referenced non-statically in `JobResourceTest`, so wanted to be consistent. I'll will be opening an epic around tests that will make sure to capture the change across all tests.